internal/provider/resource_aci_rest_managed.go

```diff
@@ -289,7 +294,12 @@ func (r *AciRestManagedResource) Read(ctx context.Context, req resource.ReadRequ
 			)
 			return
 		}
-		rn_values = strings.Split(rnMap["rn_values"], ",")
+
+		if strings.Contains(rnMap["rn_values"], "<,>") {
```

Comment:
why is this done? should we add a comment explaining this split?

Reply:
Done!